### PR TITLE
Validate DOBI

### PR DESCRIPTION
**Pull request by ThatWeb3Dev**
# Validate [{DOBI}](https://solscan.io/token/GzQzkt2B4Jr6whWVBF7XqkzWvoUy1jEd5z9tczzGg1rH)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://x.com/DobiOnSolana/status/1731903074233618516
- Coingecko/ CMC URL (If available): https://www.coingecko.com/en/coins/dobi

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [x] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.

### CODE REVIEW

**Review by ceedeepee — COMMENTED** [review]
[x] Symbol: DOBI
[x] Token decimals: 6
[x] Token icon: https://gateway.irys.xyz/099elCExuWO7iBY2_E-4mZVNLL21Hmj_nxHTGTdUxx0
[x] On-chain metadata matches
[x] No duplication on Jupiter's strict list

**Review by drakep — COMMENTED** [review]
[x] Symbol: DOBI
[x] Token decimals: 6
[x] Token icon: https://gateway.irys.xyz/099elCExuWO7iBY2_E-4mZVNLL21Hmj_nxHTGTdUxx0
[x] On-chain metadata matches
[x] No duplication on Jupiter's strict list

File: validated-tokens.csv [hunk 1]
```diff
@@ -1027,3 +1027,4 @@ BunnyBytes Golden Ticket,BBYTES,AVZvaDtfPnGbqRcsgH6eia381rPN3sExPtMs1TTnjaVq,2,h
 JITO,JTO,jtojtomepa8beP8AuQc6eXt5FriJwfFMwQx2v2f9mCL,9,https://metadata.jito.network/token/jto/image,true
 Racket,$RKT,RKT69NZHN5uovcan3q5hRbZzfJuXiGEuPmGANoBJLLz,5,https://arweave.net/Vr36ys1neJyzyfhkxv1Cxt3QJF7LhuDd_iGmvSYjLkg,true
 Leia,LEIA,7usVzynPTUJ9czdS96ezm9C6Z3hCsjb7j6TMKipURyyQ,6,https://bafkreifjempcywq7od7svwed4ds4dieemifdy4tpguba2kv5zhmapi6ufm.ipfs.nftstorage.link,true
+DOBI,DOBI,GzQzkt2B4Jr6whWVBF7XqkzWvoUy1jEd5z9tczzGg1rH,6,https://gateway.irys.xyz/099elCExuWO7iBY2_E-4mZVNLL21Hmj_nxHTGTdUxx0,true
```
Comment (00Smurf) [inline validated-tokens.csv:1030]: [x] Symbol: DOBI
[x] Token decimals: 6
[x] Token icon: https://gateway.irys.xyz/099elCExuWO7iBY2_E-4mZVNLL21Hmj_nxHTGTdUxx0
[x] On-chain metadata matches
[x] No duplication on Jupiter's strict list

**Review by 00Smurf — COMMENTED** [review]
[x] Symbol: DOBI
[x] Token decimals: 6
[x] Token icon: https://gateway.irys.xyz/099elCExuWO7iBY2_E-4mZVNLL21Hmj_nxHTGTdUxx0
[x] On-chain metadata matches
[x] No duplication on Jupiter's strict list

**Review by Timoon21 — DISMISSED** [review]

**Pull request merged**